src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php

```diff
@@ -6,6 +6,11 @@
 use Swarrot\Processor\ProcessorInterface;
 use Psr\Log\LoggerInterface;
 
+/**
+ * Class ExceptionCatcherProcessor
+ *
+ * @package Swarrot\Processor\ExceptionCatcher
```

Comment:
Done @odolbeau. Do you think I should have a look into all READMEs and messages?